File: app/src/main/java/com/wealoha/social/beans/SinaTokenBean.java
package com.wealoha.social.beans;

import com.google.gson.annotations.SerializedName;

public class SinaTokenBean extends ResultData {
	@SerializedName("access_token")
	public String accessToken;
	public String uid;
	public String name;
	@SerializedName("expires_in")
	public String expiresIn;
}

File: app/src/main/java/com/wealoha/social/beans/SinaTokenBean.kt
package com.wealoha.social.beans

import com.google.gson.annotations.SerializedName

class SinaTokenBean : ResultData() {
    @kotlin.jvm.JvmField
    @SerializedName("access_token")
    var accessToken: String? = null
    @kotlin.jvm.JvmField
    var uid: String? = null
    @kotlin.jvm.JvmField
    var name: String? = null

    @kotlin.jvm.JvmField
    @SerializedName("expires_in")
    var expiresIn: String? = null
}